api/mysagw/case/views.py

```diff
@@ -55,6 +57,19 @@ def get_queryset(self, *args, **kwargs):
             ),
         )
 
+    @action(
+        methods=["post"],
+        detail=False,
+        serializer_class=serializers.CaseTransferSerializer,
+        permission_classes=(IsAuthenticated & (IsAdmin | IsStaff),),
+        parser_classes=(JSONParser,),
+    )
+    def transfer(self, request):
```

Comment:
Would make sense to have it as a transaction to avoid mixed access if an error occurs

Reply:
Done :+1: